_layouts/blog-list.html

```diff
@@ -0,0 +1,34 @@
+---
+layout: slim
+---
+
+<section class="mb-5 py-3">
+  <div class="container">
+    <div class="row">
+      <div class="col-lg-12 mx-auto">
+        {{ content }}
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="mb-5 py-3">
+  <div class="row">
+
+    {% for post in site.posts %}
+    <div class="card border-0 col-md-6 col-lg-4">
```

Comment:
On mobile, if you're half way through the list the space between the image and the into of two separate posts is the same as the space between a post's heading and it's image. So it means it's not visually clear if the title (or image) applies to the image (or title) above or below.

Basically: some extra `mb-3` on the card would help on mobile (and shouldn't hurt on desktop).

Reply:
Done